{{cookiecutter.project_slug}}/compose/production/traefik/traefik.yml

```diff
@@ -26,9 +26,14 @@ certificatesResolvers:
         entryPoint: web
 
 http:
+  {%- set domain_dots = cookiecutter.domain_name.count('.') %}
   routers:
     web-router:
+      {%- if domain_dots == 1 or (domain_dots == 2 and '.co.' in cookiecutter.domain_name) %}
```

Comment:
I've done a bit of research, and I now believe that checking for `.co.` isn't enough. In Brasil, the `.com.br` is very common by [looking on Wikipedia](https://en.wikipedia.org/wiki/Second-level_domain), I can see there are a lot of variations across the world.

Reply:
Yeah, as I mentioned in the PR/commit this is a best effort detection. Note that the successful detection results in addition of `www` route, so we mostly err on the side of _not adding_ `www` route when the detection fails.